mautic/mautic
# Fix npm dependencies: deinstall grunt-remove, update grunt

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ x]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #12007

#### Description:
There are issues with the current javascript dependencies. This PR fixes those by removing the unused package `grunt-remove`. In addition, multiple devDependencies were updated to the latest version. These are only dev dependencies. So there is no impact for the user. 

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. run `npm install` in the root folder => there should be no error and a message in the style like this: `added 1 package, removed 15 packages, changed 14 packages, and audited 219 packages in 3s`
